src/pages/product/ProductReviews.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Review } from '../../molecules/Review'
+
+interface ProductReviewsProps {
+  reviews: any[]
+}
+
+export const ProductReviews: React.FC<ProductReviewsProps> = ({ reviews }) => {
+  return (
+    <>
```

Comment:
This is a minor comment, but sometimes you may want to wrap blocks in a container to separate it semantically. For example, in this case it may be a `div` or a `section`.